packages/storefront/src/views/Page.vue

```diff
@@ -110,4 +151,49 @@
   p-tabs-bar {
     margin-bottom: $pds-spacing-fluid-large;
   }
+
+  .skeleton {
+    height: $pds-font-line-height;
+
+    &--h1 {
+      max-width: 200px;
+      @include pds-heading-xx-large;
+      margin-top: $pds-spacing-fluid-large;
+
+      // without tabs-bar
+      &:first-child {
+        margin-top: 0;
+      }
+    }
+
+    // table of contents heading
+    &--h2 {
+      max-width: 150px;
+      @include pds-heading-medium;
+      margin: 4rem 0 1rem;
```

Comment:
I think we should work with `px` to defined spacings.

Reply:
It's to be in sync with margins of `.toc` in `TableOfContents.vue`.